cuda/matrix/fbcsr_kernels.cu

```diff
@@ -230,7 +244,19 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
 template <typename ValueType, typename IndexType>
 void fill_in_dense(std::shared_ptr<const CudaExecutor> exec,
```

Comment:
I might have missed it, but is this being tested? Is it test/matrix/matrix.cpp `ConvertToDenseIsEquivalentToRef`?

Reply:
yes